src/app.js

```diff
@@ -65,6 +65,11 @@ function listen (koaApp, config, ledgers) {
       log('app').error(error.message)
       process.exit(1)
     }
+    const ledgerUirs = Object.keys(ledgers.getLedgers())
+    for (let i = 0; i < ledgerUirs.length; i++) {
```

Comment:
This is a "low-level" way of iterating over an array; would be better to use "map"